integration/raft/channel_participation_test.go

```diff
@@ -1109,3 +1115,48 @@ func multiNodeEtcdRaftTwoChannels() *nwo.Config {
 
 	return config
 }
+
+func createJoinBlockDefineSystemChannel(channelID string) *common.Block {
+	return &common.Block{
+		Data: &common.BlockData{
+			Data: [][]byte{
+				protoutil.MarshalOrPanic(&common.Envelope{
+					Payload: protoutil.MarshalOrPanic(&common.Payload{
+						Data: protoutil.MarshalOrPanic(&common.ConfigEnvelope{
+							Config: &common.Config{
+								ChannelGroup: &common.ConfigGroup{
+									Groups: map[string]*common.ConfigGroup{
```

Comment:
do we really need all these values (hashing algorithm, orderer addresses, etc.) ?